src/api-reference/invoice/v4.purchase-request-endpoints.markdown

```diff
@@ -67,7 +79,7 @@ ToDo: Pull from documentation template.
 
 #### Request
 
-This is just a sample set of fields. The fields and values needing to be passed for your entity will vary based on your edition of SAP concur and your forms and fields setup, but should include most of these common fields.
+This is a sample set of fields. The fields and values  your entity requires will vary based on your edition of Concur Invoice, and your forms and fields configuration. This example includes commonly used fields.
```

Comment:
You've got two spaces in between 'values' and 'your. I've used the suggestion feature -- you should just be able to click a button to accept the fix.

```suggestion
This is a sample set of fields. The fields and values your entity requires will vary based on your edition of Concur Invoice, and your forms and fields configuration. This example includes commonly used fields.
```